.changeset/calm-clocks-dress.md

```diff
@@ -7,5 +7,5 @@ Tweak palette disabled token used in editable
 ```diff
 - --uitk-editable-background-disabled-low: var(--uitk-palette-interact-background-low)
 + --uitk-palette-interact-background-disabled-low
-+ --uitk-editable-background-disabled-low: var(--uitk-palette-interact-background-disabled-low)
++ --uitk-editable-tertiary-background-disabled: var(--uitk-palette-neutral-tertiary-background-disabled)
 ```
```

Comment:
@joshwooding what should we do for changesets like this?

Reply:
I believe what you've done here is fine. These changesets haven't been released so can be edited, and conflicting changesets in a release could cause confusion.

Reply:
okay cool, i need to go through the changesets and make sure all are aligned with this change :)